domains/authorization/Authorization.API/API/Authorization/Controllers/ConsentController.cs

```diff
@@ -53,9 +53,39 @@ public async Task<ActionResult> GetConsent()
         var queryResult = await mediator.Send(new GetUserOrganizationConsentsQuery(identity.Subject.ToString(), identity.OrganizationCvr!));
 
         var response = new UserOrganizationConsentsResponse(
-            queryResult.Result.Select(item => new UserOrganizationConsentsResponseItem(item.ClientName, item.ConsentDate))
+            queryResult.Result.Select(item => new UserOrganizationConsentsResponseItem(item.ClientId, item.OrganizationId, item.ClientName, item.ConsentDate))
         );
 
         return Ok(response);
     }
+
+    /// <summary>
+    /// Deletes a consent, from the organization, which the user is affiliated with.
+    /// </summary>
+    /// <param name="clientId">The ID of the client.</param>
+    /// <param name="organizationId">The ID of the organization.</param>
+    /// <returns>No content if the deletion was successful, Not Found if the consent was not found.</returns>
+    [HttpDelete("api/authorization/consents/{clientId}/{organizationId}")]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    public async Task<ActionResult> DeleteConsent([FromRoute] Guid clientId, [FromRoute] Guid organizationId)
+    {
+        var userId = identity.Subject.ToString();
+        var userOrgCvr = identity.OrganizationCvr;
+
+        try
```

Comment:
No need to catch this exception. There is a handler to return forbidden when that exception is thrown

Reply:
fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1236/commits/94b1f55c3688b382e42989a6820e5bde220f891d